src/Analysis/Ast/Impl/Analyzer/Symbols/ClassEvaluator.cs

```diff
@@ -150,9 +150,6 @@ internal sealed class ClassEvaluator : MemberEvaluator {
             // Add members from this file
             var members = Eval.CurrentScope.Variables.Where(v => v.Source == VariableSource.Declaration || v.Source == VariableSource.Import);
             _class.AddMembers(members, false);
-            // Add members from stub
```

Comment:
Why does this get removed? Shouldn't the stub be merged, or is this being done somewhere else?

Reply:
It is added in the `MergeStub` as well. Adding it here was probably a leftover from early code and caused problem - instead of merging types with appropriate comparison this piece was hard-adding methods returning `Any` removing better types we discovered.